### PR TITLE
Initial Entra ID configuration

### PR DESCRIPTION
**Pull request by az400mayrrf**
Associated with Issue #5

### CODE REVIEW

**Comment by az400mayrrf** [comment]
It doesn't look like you added any actual code?  Does this even work. There are not tests to support it.

File: ProjectClippy.csproj [hunk 1]
```diff
@@ -9,6 +9,7 @@
 
   <ItemGroup>
     <PackageReference Include="coverlet.collector" Version="6.0.0" />
+    <PackageReference Include="Microsoft.Identity.Client" Version="4.61.3" />
```
Comment (az400mayrrf) [inline ProjectClippy.csproj:12]: What code requires this assembly?

Comment (rob-foulkrod) [reply ProjectClippy.csproj:12]: Oh gosh. You are right. Let me fix that.

**Review by rob-foulkrod — APPROVED** [review]
Look good to me

**Pull request merged**